.github/workflows/publish-on-release.yml

```diff
@@ -0,0 +1,48 @@
+name: Publish Cookie Policy release
+
+# Only for testing. Should be on: release: types: [published]
+on:
+  - push
+
+jobs:
+  build:
+    name: Build Cookie Policy
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 18
+      - name: Read cookie-policy version from package.json
+        run: |
+          node -p "require('./package.json').version" > CURRENT_VERSION
+          echo "Building Cookie Policy v$(cat CURRENT_VERSION)"
+      - run: yarn install
+      - run: yarn build
+      - run: yarn test
+      - name: Show size of the build file
+        run: stat -c '%s' build/css/build.css
+      - run: cp CURRENT_VERSION build/css
+      - uses: actions/upload-artifact@v4
+        with:
+          name: css
+          path: build/css
```

Comment:
This is Vanilla folder. This would need to be cookie-policy folder (dist?).

But overall - I don't think you need to separate this into `build` and `publish-npm` jobs.
We do it in Vanilla because we also have `publish-assets` job.

I think it would be simpler if you have it all in single job, just build/test first, and then attempt publishing to npm.

Reply:
Good point, I'll merge the two together

Reply:
I've merged the file, though the CI action failed because the current version was already published. It shouldn't be a problem though.